src/exo/LoopIR_scheduling.py

```diff
@@ -2531,13 +2521,18 @@ def _make_closure(name, stmts, var_types):
 
 
 class _DoInsertPass(LoopIR_Rewrite):
-    def __init__(self, proc, stmt):
+    def __init__(self, proc, stmt, before=True):
```

Comment:
I'm guessing this is taking `before=True` because `insert_pass()` takes gaps and statements? I think it is fair to say that statement inserting primitives such as "insert pass" and "config write at" only take gaps.

Reply:
Right now nothing in `LoopIR_scheduling.py` is written to take cursors.  Therefore it's impossible to pass a gap to one of these scheduling passes.  In order to hack this situation for the time being, I'm using this additional `before` argument.  When hopefully we switch over to just passing a cursor down to these operations, then we can change this interface.